src/imitation/scripts/train_adversarial.py

```diff
@@ -36,73 +38,91 @@ def save(trainer, save_path):
       discriminator loss during generator training steps in red.
     - Plot the performance of the generator policy versus the performance of
       a random policy. Also plot the performance of an expert policy if that is
       provided in the arguments.
 
+  Ray Tune (turn on using `ray_tune_interval > 0`):
+    - Track the episode reward mean of the imitation policy by performing
+      rollouts every `ray_tune_interval` epochs.
+
   Checkpoints:
     - DiscrimNets are saved to f"{log_dir}/checkpoints/{step}/discrim/",
       where step is either the training epoch or "final".
     - Generator policies are saved to
       f"{log_dir}/checkpoints/{step}/gen_policy/".
 
   Args:
-      _seed: Random seed.
-      env_name: The environment to train in.
-      log_dir: Directory to save models and other logging to.
-      n_epochs: The number of epochs to train. Each epoch consists of
-          `n_disc_steps_per_epoch` discriminator steps followed by
-          `n_gen_steps_per_epoch` generator steps.
-      n_epochs_per_plot: An optional number, greater than or equal to 1. The
-          (possibly fractional) number of epochs between each plot. The first
-          plot is at epoch 0, after the first discrim and generator steps.
-          If `n_epochs_per_plot is None`, then don't make any plots.
-      n_disc_steps_per_epoch: The number of discriminator update steps during
-          every training epoch.
-      n_gen_plot_episodes: The number of generator update steps during every
-          generator epoch.
-      n_episodes_per_reward_data: The number of episodes averaged over when
-          calculating the average episode reward of a policy for the performance
-          plots.
-      n_episodes_eval: The number of episodes to average over when calculating
-          the average ground truth reward return of the final policy.
-      checkpoint_interval: Save the discriminator and generator models every
-          `checkpoint_interval` epochs and after training is complete. If <=0,
-          then only save weights after training is complete.
-      interactive: Figures are always saved to `output/*.png`. If `interactive`
-        is True, then also show plots as they are created.
-      expert_policy (BasePolicy or BaseRLModel, optional): If provided, then
-          also plot the performance of this expert policy.
-      init_trainer_kwargs: Keyword arguments passed to `init_trainer`,
-        used to initialize the trainer.
+    _seed: Random seed.
+    env_name: The environment to train in.
+    log_dir: Directory to save models and other logging to.
+
+    n_epochs: The number of epochs to train. Each epoch consists of
+      `n_disc_steps_per_epoch` discriminator steps followed by
+      `n_gen_steps_per_epoch` generator steps.
+    n_gen_steps_per_epoch: The number of generator update steps during every
+      generator epoch.
+    n_disc_steps_per_epoch: The number of discriminator update steps during
+      every training epoch.
```

Comment:
For consistency with above: should this be "every discriminator epoch"?

Reply:
Should be "every training epoch" for both parameters. Changed to this.